src/PrestaShopBundle/Command/ThemeCommand.php

```diff
@@ -0,0 +1,176 @@
+<?php
+/**
+ * 2007-2019 PrestaShop and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+namespace PrestaShopBundle\Command;
+
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use PrestaShop\PrestaShop\Core\Addon\Theme\ThemeManager;
+use PrestaShop\PrestaShop\Core\Addon\Theme\ThemeRepository;
+use PrestaShop\PrestaShop\Core\Addon\Theme\ThemeExporter;
+use Context;
+use Employee;
+use Shop;
+use Validate;
+
+/**
+ * Class ThemeCommand run command actions in CLI
+ */
+class ThemeCommand extends ContainerAwareCommand
+{
+    const ALLOWED_ACTIONS = array(
+        'install',
+        'enable',
+        'export',
+        'uninstall',
+    );
+
+    /**
+     * @var SymfonyStyle
+     */
+    protected $io;
+
+    /**
+     * @var int if the activation of the theme fails, return the right code
+     */
+    const RETURN_CODE_FAILED = 1;
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setName('prestashop:theme')
+            ->setDescription('Manage your theme via command line')
+            ->addArgument('action', InputArgument::REQUIRED, sprintf('Action to execute (Allowed actions: %s).', implode(' / ', self::ALLOWED_ACTIONS)))
+            ->addArgument('theme', InputArgument::REQUIRED, 'Theme or path to zip on which the action will be executed')
+            ->addArgument('shop', InputArgument::OPTIONAL, 'Shop id on which the action will be executed');
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function init(InputInterface $input, OutputInterface $output)
+    {
+        require $this->getContainer()->get('kernel')->getRootDir() . '/../config/config.inc.php';
```

Comment:
Since PR #15515 this require generate a "Notice Constant _PS_SSL_PORT_ already defined".
We can remove this line witch is take in charge on console.php head script.